crates/pera-core/src/consensus_handler.rs

```diff
@@ -355,6 +356,63 @@ impl<C: CheckpointServiceNotify + Send + Sync> ConsensusHandler<C> {
                             .stats
                             .inc_num_user_transactions(authority_index as usize);
                     }
+
+                    // If we receive a ProofMPCOutput transaction, verify that it's valid & create a system transaction
+                    // to store its output on the blockchain, so it will be available for the initiating user.
```

Comment:
The naming here is confusing because "ProofMPCOutput transaction" makes it sound like it has already happened inside the
Move VM. It's more like there is a ProofMPCOutput sent by one Validator who then locally creates a Move System Tx if the
ProofMPCOutput message contains a correct output.

Reply:
which name would make more sense for you?

Reply:
Proof will be deleted on the next PR, no point to waste time on it.